qtawesome/iconic_font.py

```diff
@@ -173,7 +173,47 @@ def _paint_icon(self, iconic, painter, rect, mode, state, options):
 
         painter.setOpacity(options.get('opacity', 1.0))
 
-        painter.drawText(rect, int(Qt.AlignCenter | Qt.AlignVCenter), char)
+        draw = options.get('draw', 'auto')
+        if draw == 'text':
+            draw_path = False
+        elif draw == 'path':
+            draw_path = True
+        else:
+            # Use QPainterPath when animation
+            # to fix tremulous spinning icons #39
```

Comment:
Improve a little bit the comment here
```suggestion
            # Use QPainterPath when setting an animation
            # to fix tremulous spinning icons.
            # See #39
```